benchkit/hdc/__init__.py

```diff
@@ -0,0 +1,219 @@
+# SPDX-License-Identifier: MIT
+"""
+Module to handle hdc (harmony debug bridge) interactions between host and remote phone.
+"""
+from typing import Iterable, Optional
+
+from benchkit.shell.shell import get_args, shell_out
+from benchkit.utils.types import Command, PathType
+
+
+def _identifier_from(ip_addr: str, port: int) -> str:
+    return f"{ip_addr}:{port}"
+
+
+class HDCError(Exception):
+    """Handle for errors from adb."""
+
+
+class HDCDevice:
+    """Representation of a device connected through hdc."""
+
+    def __init__(
```

Comment:
OH devices can be identfied either by `ip:port` or alternatively by the serial number of the device. The latter is still missing here.

Reply:
are we implementing that in this PR? @TriedAngle